Fixing the visual issues for the toolbar and sidebar

Manarnz opened the pull request:
Previously, the config sidebar would not scroll down. Instead, it would keep going downwards everytime a new request is made. It is not exactly issue #477, as the issue changed once the "Started Configurations Revamp in new branch" PR was mereged.

Manarnz committed:
Fixed issue #744

Previously, the config sidebar would not scroll down. Instead, it would keep going downwards everytime a new request is made. It is not exactly issue #744, as the issue changed once the "Started Configurations Revamp in new branch" PR was mereged.

leshghi reviewed (changes requested):
I'm not sure what Issue you're working on - is this supposed to be issue #477 that you were assigned? The problem is actually that the logo hides the configurations even after you scroll down. Scrolling has already been enabled for the configs. It would be great to bring the div that holds the config sidebar up above the logo somehow so there is no overlap, while also extending the length of the overall side panel so that when you scroll down on the webpage as a whole there is no gap. See picture in issue for reference.

Manarnz commented:
> I'm not sure what Issue you're working on - is this supposed to be issue #477 that you were assigned? The problem is actually that the logo hides the configurations even after you scroll down. Scrolling has already been enabled for the sidebar. It would be great to bring the div that holds the config sidebar up above the logo somehow so there is no overlap.

After Angela and I merged our PR, the configInspector would not scroll down. Instead it just kept incrementing downwards as demonstrated in the photo. 
![image](https://user-images.githubusercontent.com/83615524/125092562-77d80600-e09f-11eb-8e00-5838f999d1f7.png)
I fixed this issue in this PR. This is why I stated that it is not exactly issue #477. I will now work on the logo problem, thank you for the clarification!

leshghi commented:
> > I'm not sure what Issue you're working on - is this supposed to be issue #477 that you were assigned? The problem is actually that the logo hides the configurations even after you scroll down. Scrolling has already been enabled for the sidebar. It would be great to bring the div that holds the config sidebar up above the logo somehow so there is no overlap.
> 
> After Angela and I merged our PR, the configInspector would not scroll down. Instead it just kept incrementing downwards as demonstrated in the photo.
> ![image](https://user-images.githubusercontent.com/83615524/125092562-77d80600-e09f-11eb-8e00-5838f999d1f7.png)
> I fixed this issue in this PR. This is why I stated that it is not exactly issue #477. I will now work on the logo problem, thank you for the clarification!

Ah got it - In that case can you make an actual issue for that as well and tag it in this PR? If possible it would be awesome if you could work on #477 on this PR as well since the two are related.

leshghi commented:
> > > I'm not sure what Issue you're working on - is this supposed to be issue #477 that you were assigned? The problem is actually that the logo hides the configurations even after you scroll down. Scrolling has already been enabled for the sidebar. It would be great to bring the div that holds the config sidebar up above the logo somehow so there is no overlap.
> > 
> > 
> > After Angela and I merged our PR, the configInspector would not scroll down. Instead it just kept incrementing downwards as demonstrated in the photo.
> > ![image](https://user-images.githubusercontent.com/83615524/125092562-77d80600-e09f-11eb-8e00-5838f999d1f7.png)
> > I fixed this issue in this PR. This is why I stated that it is not exactly issue #477. I will now work on the logo problem, thank you for the clarification!
> 
> Ah got it - In that case can you make an actual issue for that as well and tag it in this PR? If possible it would be awesome if you could work on #477 on this PR as well since the two are related.

Also - there is no issue 744, so could you please remove the references to that in this PR? Thanks!

Manarnz committed:
Fixed the position of the toolbar and title logo and a tempoary fix to the logo overlapping issue.

To fix the issue with the logo overlapping the configs, I made configInspector a little bit shorter. I am not sure if this is a good solution or not. I also decided to keep the logo "BloomingLeaf" seperate from the toolbar. I fixed its position as well as the toolbar.

leshghi reviewed (changes requested):
Have you tested this? The configs still overlap with the logo. This might involve a larger css fix/re-arrangement

Manarnz committed:
Merge branch 'backbone-refactor' of https://github.com/amgrubb/BloomingLeaf into maac-issue477

Manarnz committed:
Made .inspector and .attribuation disappear in analysismode

Manarnz committed:
Removed toolbar shadow

amgrubb committed:
Changed the position of the attribution to be relative.

amgrubb committed:
Merge branch 'backbone-refactor' into maac-issue477

Manarnz committed:
Fixed the placement of BloomingLeaf label

I split the text from the box style.css so I could easily change the position of the text. I am wondering if it is aceptable to manually calculate the position or to use percentage (which is a cleaner practice).

Manarnz committed:
Created space between configs and fixed some minor issues

Manarnz committed:
Moved btn-svg to save dropdown

Manarnz committed:
Simple changes to the layout

Removed the label for view mode
Changed the size of paper to fit modelling view
Fixed the length difference between the two sidebars

Manarnz committed:
Change the order of buttons in toolbar

Manarnz committed:
Fixed the issue with EVO placement

Manarnz committed:
Changed the toolbar

Manarnz committed:
Merge branch 'backbone-refactor' of https://github.com/amgrubb/BloomingLeaf into maac-issue477

YesugenB commented:
When you go to Analysis view, the Attribution section disappears, and is not there when you go back to Modeling view.

Manarnz committed:
Update index.html

Manarnz commented:
> When you go to Analysis view, the Attribution section disappears, and is not there when you go back to Modeling view.

It is supposed to disappear in analysis mode. As for the second part, for some reason it is working for me. Could you please check now?

YesugenB commented:
Yup it's fixed now! Thanks

YesugenB commented on leaf-ui/css/style.css:
```diff
@@ -172,7 +169,7 @@ body.waiting *
    color: black;
    text-shadow: none;
-   box-shadow: inset 0 0 0 1px rgba(0,0,0,0.1),0px 0 0 1px rgba(255,255,255,0.1);
+   /* box-shadow: inset 0 0 0 1px rgba(0,0,0,0.1),0px 0 0 1px rgba(255,255,255,0.1); */
```
Can this be removed now?

YesugenB commented on leaf-ui/index.html:
```diff
@@ -80,39 +77,45 @@
         </div>
 
+        <button id="analysis-btn" class="green-btn btn model-only" style="border-radius:none;">Analysis</button>
+        <button id="modeling-btn" class="red-btn btn analysis-only" style='display: none';>Modeling</button>
```
Can we add `""` around `style` instead of `''` so it's consistent

YesugenB replied:
And `none';` should be `none;'`?

YesugenB commented on leaf-ui/index.html:
```diff
@@ -80,39 +77,45 @@
-            style="position:relative; border-style:none none none solid; border-color: rgb(74, 105, 245); border-width: 1px">Intermediate
+        <button id="btn-view-intermediate" class="btn model-only">Intermediate
             Values</button>
         <!--This is the modal box of Intemdiate Values-->
```
`Intermediate`

YesugenB commented on leaf-ui/index.html:
```diff
@@ -80,39 +77,45 @@
-        <button id="analysis-btn" class="green-btn btn model-only" style="border-radius:none;">Analysis</button>
-
-        <button id="simulate-single-path-btn" ; class="dark-green-btn btn analysis-only" ;
+        <button id="simulate-single-path-btn" ; class="btn analysis-only" ;
```
I'm not sure that we need the` ;` between id and class in this and line 111.

YesugenB commented on leaf-ui/index.html:
```diff
@@ -80,39 +77,45 @@
             style="border-radius: none; display: none" ;>Explore States</button>
-        <button id="modeling-btn" ; class="red-btn btn analysis-only"
-            style='display: none';>Modeling</button>
+       <!-- <div id="dropdown-model" class="dropdown analysis-only" style='display: none'>
```
Change quotation marks around `style`

Manarnz replied:
This is commented out.

YesugenB reviewed (approved):
Looks good

amgrubb committed:
small change to element inspector

amgrubb committed:
Merge branch 'backbone-refactor' into maac-issue477

amgrubb committed:
add back in right inspector

amgrubb reviewed (approved):
Nice work, it looks great! Please address @YesugenB's comments.

amgrubb reviewed (changes requested):
Sorry, clicked the wrong button. Please address @YesugenB's comment.

Manarnz committed:
Implemented comments

Manarnz committed:
Merge branch 'maac-issue477' of https://github.com/amgrubb/BloomingLeaf into maac-issue477

amgrubb reviewed (approved):
Excellent, thanks for making these changes.
